change-address/REPWRITERSPECS/BANNO.CHANGE.ADDR.V1.POW

```diff
@@ -1323,6 +1431,7 @@ PROCEDURE INITIALIZEDATA
 
  TESTUSER=FALSE
  PERFORMMATCHING=FALSE
+ PARAMUNALLOWEDCHARS="NONE"
```

Comment:
Do you need this init here since this variable is also init'd in READCONFIGFILESETTINGS?

Reply:
Done